exercise_08/formatStringByWordsLength.js

```diff
@@ -16,3 +16,20 @@
  * CASE #2 * input: "This is short one."
  * output: "this is short one."
  */
+
+/**
+ * 
+ * @param {string} sentence 
+ * @returns {string}
+ */
+function formatStringByWordsLength(sentence) {
+   const words = sentence.split(' ');
+   if (words.length > 5) {
```

Comment:
Avoid magic numbers. 

```suggestion
   if (words.length > THRESHOLD) {}
```

- [ ] Extract number to constant